datamodel/changelogs/0001/04_vsa_kek_extension.sql

```diff
@@ -169,7 +169,7 @@ COMMENT ON COLUMN tww_od.data_media.identifier IS 'yyy_Name des Datenträgers. B
 COMMENT ON COLUMN tww_od.data_media.kind IS 'Describes the type of data media / Beschreibt die Art des Datenträgers / Décrit le genre de support de données';
  ALTER TABLE tww_od.data_media ADD COLUMN location  varchar(50) ;
 COMMENT ON COLUMN tww_od.data_media.location IS 'Location of the data medium / Ort, wo sich der Datenträger befindet / Emplacement du support de données';
- ALTER TABLE tww_od.data_media ADD COLUMN path  varchar(1023) ;
+ ALTER TABLE tww_od.data_media ADD COLUMN path TEXT;
```

Comment:
Can you explain this change? This field should be not longer than 1023 - else it is not a valid URI anymore.

Reply:
For such large fields, I would be in favor of having a constraint (db or client) instead. It is then easier for the user to understand why it is blocking and give the user some context. 

Using a db defined field size could in some cases just truncate their content.

But if that's an issue I can live with both since there's not so much difference :
https://stackoverflow.com/questions/4848964/difference-between-text-and-varchar-character-varying